### PR TITLE
Missing };

### PR DESCRIPTION
**Pull request by EGI-ILM**
I have not read the code, just placed it according to the indentation. Please VERIFY that this is located in the right place.

### CODE REVIEW

File: static/js/egi.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ jQuery(document).ready(function () {
           $('.progress').stop(true, false, true).fadeToggle(100);
       });
   }
+};
```
Comment (gwarf) [inline static/js/egi.js:68]:
```suggestion
});
```
Apparently there is also a missing `)`.
That said I'm not sure what's the exact purpose of those changes, if they are still required, and what may not be correctly working if the current version is broken.

@thebe14 I think you are the one that added this, any thoughts?

**Comment by gwarf** [comment]
@enolfc , @andrea-manzi what do you think of this one?

**Comment by andrea-manzi** [comment]
can we deploy and see what is changing?

**Comment by gwarf** [comment]
> can we deploy and see what is changing?

I've just enabled the preview.
From my local tests, it seemed that the change I proposed was necessary to resolve warnings/errors from the dev tools.
Didn't saw much changes in the UI, but wasn't sure what may be broken or not.

File: static/js/egi.js (hunk 1 above)
Comment (thebe14) [reply static/js/egi.js:68]: There is definitely missing `});` although I am not clear how I could commit that and still have the things work.

**Comment by thebe14** [comment]
Commit it with the `});` added, I will start working on the documentation next week, will check that the category icon styles are properly applied in the sidebar.

**Comment by gwarf** [comment]
OK, thanks Levente, will merge this once the tests are green.
You purposely disabled linting those files in #244, but it may have caught this ;)

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**